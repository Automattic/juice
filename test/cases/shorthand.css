p {
  padding-bottom: 10px;
}

#p {
  padding: 0px;
}
