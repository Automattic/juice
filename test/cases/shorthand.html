<html><body>
<p id="p">woot</p>
</body></html>
