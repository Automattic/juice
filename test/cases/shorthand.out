<html><body>
<p id="p" style="padding: 0px;">woot</p>
</body></html>
